types/src/genesis/mod.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::SectorSize;
+use address::Address;
+use num_bigint::bigint_ser;
+use serde::Serialize;
+use vm::TokenAmount;
+
+#[derive(Serialize)]
+enum ActorType {
+    // Account,
+// MultiSig,
+}
+
+#[derive(Serialize)]
+pub struct Actor {
+    actor_type: ActorType,
+    #[serde(with = "bigint_ser")]
+    token_amount: TokenAmount,
+}
+
+#[derive(Serialize)]
+pub struct Miner {
```

Comment:
can you add pascal casing to these as well?

Reply:
added